website/react-magma-docs/src/pages/api-intro/page-templates.mdx

```diff
@@ -15,56 +15,117 @@ This template is ideal for applications that need as much of the screen as possi
   <img src="../../images/layout/template-full-width.png" alt=" " />
 </figure>
 
-```noRender
+```tsx
 import React from 'react';
-import styled from '@emotion/styled';
 import { Header } from '@cengage-patterns/header';
-import { magma, useMediaQuery } from 'react-magma-dom';
-
-const Container = styled.div`
-  display: grid;
-  grid-template-columns: auto 1fr;
-  grid-template-rows: auto 1fr;
-  grid-template-areas:
-    'header header'
-    'content content';
-  gap: 2px;
-`;
-
-const StyledHeader = styled(Header)`
-  grid-area: header;
-`;
-
-const Content = styled.div`
-  grid-area: content;
-  max-width: 1600px;
-  width: calc(100% - 48px);
-  justify-self:center;
-  background:${magma.colors.neutral300};
-  padding:${magma.spaceScale.spacing05};
-  height:calc(100vh - 122px);
-`;
-
-const Sidebar = () => (
-  <StyledSidebar>Sidebar</StyledSidebar>
-);
+import {
+  Drawer,
+  DrawerPosition,
+  Grid,
+  GridAlignContent,
+  GridItem,
+  GridJustifyContent,
+  magma,
+  useMediaQuery,
+} from 'react-magma-dom';
 
 export function Example() {
+  const [showDrawer, setShowDrawer] = React.useState(false);
 
   const isSmallerScreen = useMediaQuery(
     `(max-width:${magma.breakpoints.small}px)`
   );
 
   return (
-
-      <Container>
-
-        <StyledHeader isInverse logo={<strong>LOGO</strong>} breakpoint={magma.breakpoints.large} isCompact={isSmallerScreen} />
-
-        <Content>Content</Content>
-
-      </Container>
-
+    <Grid
+      GridJustifyContent={GridJustifyContent.center}
```

Comment:
```suggestion
      gridJustifyContent={GridJustifyContent.center}
```

Reply:
According to the documentation: 


![image](https://user-images.githubusercontent.com/16470877/191348045-346c13af-c322-40cb-824e-444e5fde208e.png)